demo/build.gradle

```diff
@@ -0,0 +1,27 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+}
+
+apply from:  rootProject.file("$rootDir/common.gradle")
+
+android {
+    namespace 'com.uid2.demo'
+
+    defaultConfig {
+        applicationId "com.uid2.demo"
+        versionCode 1
+        versionName "1.0"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
```

Comment:
I suggest we EXPECT people to integrate Proguard and supply a Proguard configuration if needed.

Reply:
Yup, definitely. Will take a note to make sure we integrate Proguard (and supply config) once library is closer to being completed.